src/java.base/share/classes/java/net/InetAddress.java

```diff
@@ -785,9 +785,15 @@ String getHostName(boolean check) {
     }
 
     /**
-     * Gets the fully qualified domain name for this IP address.
-     * Best effort method, meaning we may not be able to return
-     * the FQDN depending on the underlying system configuration.
+     * Gets the fully qualified domain name for this
+     * {@linkplain InetAddress#getAddress() IP address} using the system-wide
+     * {@linkplain InetAddressResolver resolver}.
+     *
+     * <p>The system-wide resolver will be used to do a reverse name lookup of the IP address.
```

Comment:
The wording in this paragraph seems a bit different to getHostName and I think we should try to use the same terms in both methods if possible, e.g. getHostName talks about the "system configured resolver" where here it uses "system-wide resolve". We have have "used" vs. "performed". So maybe look at the methods again as I think we should be a bit more consistent with the wording if we can.

Reply:
Hello Alan, except for a couple of places in `InetAddress` javadoc, the rest of the javadoc (and code comments) in `InetAddress` use the term system-wide resolver instead of system configured resolver. Furthermore, the java.net.spi.InetAddressResolver itself uses the system-wide resolver term. So I've updated these two places in InetAddress to replace the use of "system configured resolver" to "system-wide resolver".